components/icu/src/lib.rs

```diff
@@ -426,6 +426,65 @@ pub mod properties {
     //! assert_eq!(script.get('木' as u32), Script::Han);  // U+6728
     //! ```
     //!
+    //! ## Property data for `Script` and `Script_Extensions`
+    //!
+    //! ```
```

Comment:
Suggestion (optional): Make this example more concise; you don't need all the edge cases.  The example on the specific function can have all the edge cases.

Reply:
Done.